easybuild/easyconfigs/f/fenics-basix/fenics-basix-0.9.0-foss-2023b.eb

```
@@ -0,0 +1,34 @@
+name = 'fenics-basix'
+homepage = 'https://github.com/FEniCS/basix'
+description = "Basix is a finite element definition and tabulation runtime library"
+
+versionsuffix = ''
+
+easyblock = 'CMakeMake'
+
+# Version and source
+version = '0.9.0'
+source_urls = ['https://github.com/FEniCS/basix/archive']
+sources = ['v%(version)s.tar.gz']
+
+# Toolchain (adjust to match your target system)
+toolchain = {'name': 'GCC', 'version': "13.2.0"}
```

```suggestion
toolchain = {'name': 'gfbf', 'version': "2023b"}
```

The toolchain already provides FlexiBLAS 😄 
You then also need to change the filename to `fenics-basix-0.9.0-gfbf-2023b.eb`.